deps/lib/README.txt

```diff
@@ -0,0 +1,26 @@
+Basis set exchange: Basis set bundle
```

Comment:
This file must be included in your `.gitignore` list. It is one of those files that we don't need to push into GitHub.